docs/README.md

```diff
@@ -65,6 +65,65 @@ see [Jupyter docs](https://jupyter-client.readthedocs.io/en/stable/kernels.html#
 
 Uninstall: `./gradlew uninstall`
 
+## Updating
```

Comment:
Please run move it to README_STUB.md and run `./gradlew generateReadme`

Reply:
done